doc/index.md

```diff
@@ -40,10 +40,17 @@ Use "distrobuilder [command] --help" for more information about a command.
 
 In the following, we see how to create a container image for LXD.
 
-### Installation
+## Installing from package
 
-Currently, there are no binary packages of `distrobuilder`. Therefore, you will need to compile it from source.
-To do so, first install the Go programming language, and some other dependencies.
+`distrobuilder` is available from the snapstore.
+
+```
+snap install distrobuilder
```

Comment:
Should be `snap install distrobuilder --classic`

Reply:
Which currently won't quite work as there's nothing in the stable channel, but we're working on fixing that now that we have the 1.0 release.

Reply:
Right, I just confirmed that `snap install --edge distrobuilder --classic` does the right thing for now.